### PR TITLE
Updating the LogicAppTemplate for IDPS Signatures

### PR DESCRIPTION
**Pull request by shabaz-github**
Updating the LogicAppTemplate for IDPS Signatures

### CODE REVIEW

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json [hunk 1]
```diff
@@ -190,7 +223,7 @@
                                     {
                                         "name": "email",
                                         "type": "string",
-                                        "value": "[parameters('Email')]"
+                                        "value": "[parameters('Recipient_Address')]"
```
Comment (tobystic) [inline Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:226]: Recipient Email-address:

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/README.md [hunk 2]
```diff
@@ -1,2 +1 @@
 Deploy this template to create Logic APP for notifying us on new IDPS signature updates.
```
Comment (tobystic) [inline Azure Firewall/Template - Logic app for IDPS signature updates notification/README.md:1]: Please reword:

_Deploy this template to create a Logic App to receive notifications on new IDPS signature updates for Azure firewall_

Is the Deploy button implemented yet? if not, we cannot use the word "Deploy"

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json [hunk 3]
```diff
@@ -3,41 +3,74 @@
...
             "type": "String"
         },
-        "connections_office365_1_externalid": {
-            "defaultValue": "/subscriptions/****/resourceGroups/****/providers/Microsoft.Web/connections/office365-1",
-            "type": "String"
+"Sender_Address": {
+            "defaultValue": "",
+            "type": "String",
+  "metadata": {
+                "displayName": "Enter the sender's email address to form the connection to O365 Outlook",
```
Comment (tobystic) [inline Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:13]: Display Name: "Enter sender's email address"

Description:  "Enter the sender's email address to form the connection to O365 Outlook"

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json [hunk 4]
```diff
@@ -3,41 +3,74 @@
...
+                "description": "Enter the sender's email address to form the connection to O365 Outlook"
+                }
         },
-		"Email": {
+		"Recipient_Address": {
             "defaultValue": "",
-            "type": "String"
+            "type": "String",
+ "metadata": {
+                "displayName": "Enter the email address to send the IDPS Rules update notification to",
```
Comment (tobystic) [inline Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:21]: "displayName": "Enter recipient email address to receive new signature updates",
                "description": "Enter the email address to send the IDPS Signature update notification"

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json [hunk 5]
```diff
@@ -3,41 +3,74 @@
...
+                "description": "Enter the email address to send the IDPS Rules update notification to"
+                }
         },
 		"Subscription_ID": {
             "defaultValue": "",
-            "type": "String"
+            "type": "String",
+ "metadata": {
+                "displayName": "Enter the Subscription Id of where the Firewall Policy is located",
```
Comment (tobystic) [inline Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:29]: Subscription ID

**Review by tobystic — CHANGES_REQUESTED** [review]
Shabaz,
Please look at the comments and review

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (shabaz-github) [reply Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:29]: Looks good

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json (hunk 1 above)
Comment (shabaz-github) [reply Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:226]: Looks good

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json (hunk 3 above)
Comment (shabaz-github) [reply Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:13]: Looks good

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json (hunk 4 above)
Comment (shabaz-github) [reply Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:21]: Looks good

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json (hunk 5 above)
Comment (shabaz-github) [reply Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json:29]: Looks good

File: Azure Firewall/Template - Logic app for IDPS signature updates notification/README.md (hunk 2 above)
Comment (shabaz-github) [reply Azure Firewall/Template - Logic app for IDPS signature updates notification/README.md:1]: The deploy button is not created yet - For now we can say  -  "Use this template"

Comment (shabaz-github) [reply Azure Firewall/Template - Logic app for IDPS signature updates notification/README.md]: Made the requested change

**Pull request merged**